src/App.js

```diff
@@ -90,6 +102,8 @@ class App extends Component {
           handleSong={this.addSong} 
           buttonIcon={[<RiHeartAddLine className="handle-song-icon"/> , <FaHeart className="heart-icon" data-cy="heart-icon"/>]}
           />}
+          {this.state.error &&
+          <h2>{this.state.error}</h2>}
```

Comment:
Might be small considering its just twice but maybe put this into a helper function?